metricbeat/module/zookeeper/mntr/_meta/data.json

```diff
@@ -1,14 +1,21 @@
 {
     "@timestamp": "2017-10-12T08:05:34.853Z",
-    "beat": {
+    "agent": {
         "hostname": "host.example.com",
         "name": "host.example.com"
     },
+    "event": {
+        "dataset": "zookeeper.mntr",
+        "duration": 115000,
+        "module": "zookeeper"
+    },
     "metricset": {
-        "host": "zookeeper:2181",
-        "module": "zookeeper",
-        "name": "mntr",
-        "rtt": 115
+        "name": "mntr"
+    },
+    "service": {
+        "address": "localhost:2181",
+        "type": "zookeeper",
+        "version": "3.4.8--1, built on 02/06/2016 03:18 GMT"
```

Comment:
Oh jeez 🤦‍♂️ 

Well, I didn't think our `keyword` `version` field would get versions that detailed 😆I guess that's a testament to the flexibility of ECS

Reply:
The nice part is that 3.4.8 is at the benning, so prefix queries for 3.4 still work.